[Mini] Docs info for blocking_factor (#1096)
ECP-WarpX/WarpX: Docs/source/running_cpp/parallelization.rst

AMReX ``max_grid_size`` and ``blocking_factor``
-----------------------------------------------

* ``amr.max_grid_size`` is the maximum number of cells per **grid** along each
  direction (default ``amr.max_grid_size=32`` in 3D).

* ``amr.blocking_factor``: is the minimum number of cells per **grid** along each
  direction (default ``amr.blocking_factor=8``).

>   Note that the ``max_grid_size`` also has to be divisible by ``blocking_factor``.

Reviewer: I think what this tried to express was also:
```suggestion
  Note that both the domain (at each level) and ``max_grid_size`` must be divisible by ``blocking_factor``.
```
